blak05-cv04

krystofblak05 opened the pull request:
https://eso.vse.cz/~blak05/cv04/

krystofblak05 committed:
Blak05-cv04

https://eso.vse.cz/~blak05/cv04/

nvbach91 commented on www/blak05/cv04/registration.php:
```diff
@@ -0,0 +1,136 @@
+            <form method="POST" action="<?php $_SERVER['PHP_SELF'];?>"> 
+                <label>Name:</label>
+                <input name="name"><br>
+                <label>Email:</label>
```
chce to zapamatovat vyplnene udaje ve formulari

nvbach91 commented on www/blak05/cv04/registration.php:
```diff
@@ -0,0 +1,136 @@
+         <h1>$subject</h1>
+         <p>$message</p>
+     ";
+     return mail($email, $subject, $body, $header);
```
headery musite implodovat do stringu

nvbach91 commented on www/blak05/cv04/registration.php:
```diff
@@ -0,0 +1,136 @@
+        $newRecord = implode(';', $userInfo) . "\r\n";
+
+    //vložit do souboru
+        file_put_contents($databaseFileName, $newRecord, FILE_APPEND);
```
mozna nemate nastavene pravo na zapis do souboru, nejde se mi prihlasit

krystofblak05 replied:
celé původní cv04 jsem smazal z esa a nahrál verzi, kterou jsem před minutou nahrál sem. A db opět nefungovala a bylo nutné ve filezille dát právo na zápis. Ale než jsem to uploadoval, tak jsem práva nastavil, nevím, jak bych to měl udělat, aby i při uploadu dával soubor práva pro zápis každému. Takže je možné, že chyba se u vás zase bude opakovat.

krystofblak05 committed:
blak05-cv04

Upravil jsem formuláře, at drží data, upravil jsem mail header a znovu jsem uložil db, tak aby se dalo zapisovat. Lokálně mi to funguje dobře, kdyby byl stále problém, tak bych to musel řešit jinak.
